src/components/appointment/AppointmentHostCard.tsx

```diff
@@ -0,0 +1,98 @@
+            <AppointmentHeader detailHeader={props.appointmentDetail.detailHeader} isHost={true} />
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Image</Typography>
+              <Grid container spacing={1}>
+                {props.appointmentDetail.images.map((e, index) => (
+                  <Grid item key={'i' + index}>
+                    <Image src={e} alt="location" width={150} height={150} />
+                  </Grid>
+                ))}
+              </Grid>
+            </Box>
+          </Stack>
+        </CardContent>
+      </BorderWithShadow>
+    </Grid>
+    <Grid item xs={12} md={6} style={{ display: "flex" }} >
+      <BorderWithShadow>
+        <CardContent >
+          <Stack spacing="12px" sx={{ padding: "20px", height: "100%" }}>
+            <Typography variant="h5" sx={{ fontWeight: "bold" }}>Number of Participant : {props.appointmentDetail.participantAmount}</Typography>
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Pending Participant List :</Typography>
+              <Grid container spacing={1}>
+                {props.appointmentDetail.pendingParticipants.map((e, index) => (
+                  <Grid item key={'p' + index}>
+                    <Chip
+                      label={e}
+                      variant="outlined"
+                      color='primary'
+                      style={{ ...chipsStyle, backgroundColor: '#9797971A' }}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </Box>
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Accept Participant List :</Typography>
+              <Grid container spacing={1}>
+                {props.appointmentDetail.acceptParticipants.map((e, index) => (
+                  <Grid item key={'a' + index}>
+                    <Chip
+                      label={e}
+                      variant="outlined"
+                      color="success"
+                      style={{ ...chipsStyle, backgroundColor: '#42FF001A' }}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </Box>
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Reject Participant List :</Typography>
+              <Grid container spacing={1}>
+                {props.appointmentDetail.rejectParticipants.map((e, index) => (
+                  <Grid item key={'r' + index}>
+                    <Chip
+                      label={e}
+                      variant="outlined"
+                      color='error'
+                      style={{ ...chipsStyle, backgroundColor: '#FF00001A' }}
+                    />
+                  </Grid>
+                ))}
+              </Grid>
+            </Box>
+          </Stack>
```

Comment:
1. `Number of Participant` should be at least 1 since initially create the appointment since `host` is counted as participant
2. Initially, `Pending Participant List` should contain at least 1 participant (which is not `host`) since if user didn't add any participant when create appointment, the error would occur indicating that add participant should not be empty.
3. `Accept Participant List` should initially contain `host` but in the code seems like the text is missing.
<img width="354" alt="image" src="https://user-images.githubusercontent.com/62459518/222941627-ea3da1fa-baa4-4fa4-9079-5033dc1f5eed.png">

Reply:
This is backend error. Should be fixed now